dart/lib/src/scope.dart

```diff
@@ -81,7 +81,11 @@ class Scope {
 
   final SentryOptions _options;
 
-  Scope(this._options) : assert(_options != null, 'SentryOptions is required');
+  Scope(this._options) {
+    if (_options == null) {
+      throw ArgumentError('SentryOptions is required');
```

Comment:
u were using `ArgumentError.notNull` but now `ArgumentError(message)`, can we chose one approach and go for it?

Reply:
✅ I just replaced the last ArgumentError.notNull(name) by ArgumentError(message)